experiments/ClimaEarth/components/atmosphere/climaatmos.jl

```diff
@@ -425,14 +425,11 @@ function get_atmos_config_dict(coupler_dict::Dict, job_id::String, atmos_output_
 
     # use atmos toml if coupler toml is not defined
     atmos_toml = joinpath.(pkgdir(CA), atmos_config["toml"])
-    coupler_toml = joinpath.(pkgdir(ClimaCoupler), coupler_dict["coupler_toml"])
+    coupler_toml = coupler_dict["coupler_toml"]
```

Comment:
This imposes the requirement that the filepaths in `coupler_toml` need to be relative to where the simulation is being run from. Is that something that'll be useful to you in calibration? If you don't need it, I slightly prefer to keep the previous behavior where tomls are expected to be specified as a path within `ClimaCoupler.jl/`

Reply:
This change turned out to be unnecessary, thanks for catching it. I'll revert this line and update the NEWS